torch/_inductor/decomposition.py

```diff
@@ -404,6 +405,39 @@ def dequantize_per_tensor_tensor_decomp_impl(
     return (input.to(torch.float32) - zero_point) * scale
 
 
+@register_decomposition([aten.grid_sampler_2d])
+@pw_cast_for_opmath
+def grid_sampler_2d(
+    a: torch.Tensor,
+    grid: torch.Tensor,
+    interpolation_mode: int = 0,
+    padding_mode: int = 0,
+    align_corners: bool = False,
+) -> torch.Tensor:
+
+    # We do not expand the grid (_expand_grid=False) on cpu for performance reasons
```

Comment:
Again, leave an extended note here or in the decomp about what are these "performance reasons"